nuget/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <!-- Workaround for https://github.com/dotnet/roslyn/issues/4655 -->
+  <ItemGroup Condition="Exists('$(MSBuildProjectDirectory)\AnalyzerReleases.Shipped.md')" >
+	  <AdditionalFiles Include="AnalyzerReleases.Shipped.md" />
+  </ItemGroup>
+  <ItemGroup Condition="Exists('$(MSBuildProjectDirectory)\AnalyzerReleases.Unshipped.md')" >
+	  <AdditionalFiles Include="AnalyzerReleases.Unshipped.md" />
+  </ItemGroup>
```

Comment:
We should just leave this out. Instead of having the targets file, put a comment at the top of the generated file indicating that the user needs to manually include it in AdditionalFiles.

Reply:
I prefer that to add this tooling support so the user does not have to grok though extra comment and then perform manual steps to achieve the exact same thing. There is no harm in having an additional targets to achieve this workaround. We can delete it once https://github.com/dotnet/roslyn/issues/4655 is fixed.